staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -256,5 +275,8 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"during this window all incoming requests will be rejected with a status code 429 and a 'Retry-After' response header, "+
 		"in addition 'Connection: close' response header is set in order to tear down the TCP connection when idle.")
 
+	fs.StringSliceVar(&s.DisableCompressionForClientIPs, "disable-compression-for-client-ips", s.DisableCompressionForClientIPs, ""+
+		"Comma separated list of CIDR IP ranges (parsable by net.ParseCIDR, as defined in RFC 4632 and RFC 4291) for which traffic compression should be disabled.")
```

Comment:
can you give the syntax here rather than referencing a golang func?

Reply:
I added example and referenced the standards describing the notation. Is it OK?